jdaviz/components/viewer_data_select_item.vue

```diff
@@ -22,7 +22,7 @@
       </j-tooltip>
     </div>
 
-    <j-tooltip :tooltipcontent="'data label: '+item.name" span_style="font-size: 12pt; padding-top: 6px; padding-left: 6px; width: calc(100% - 140px); white-space: nowrap; cursor: default;">
+    <j-tooltip :tooltipcontent="'data label: '+item.name" span_style="font-size: 12pt; padding-top: 6px; padding-left: 6px; width: calc(100% - 80px); white-space: nowrap; cursor: default;">
```

Comment:
this fixes an alignment issue with the "x" button.  #1491 apparently didn't properly set the alignment for all cases.